pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+// InterPodAffinityArgs holds arguments used to configure the InterPodAffinity plugin.
+type InterPodAffinityArgs struct {
+	// HardPodAffinityWeight is the scoring weight for existing pods with a
+	// matching hard affinity to the incoming pod.
+	HardPodAffinityWeight int32
+}
+
+// NodeLabelArgs holds arguments that used to configure the NodeLabel plugin.
+type NodeLabelArgs struct {
+	// PresentLabels should be present for the node to be considered a fit for hosting the pod
+	PresentLabels []string
+	// AbsentLabels should be absent for the node to be considered a fit for hosting the pod
+	AbsentLabels []string
+	// Nodes that have labels in the list will get a higher score.
+	PresentLabelsPreference []string
+	// Nodes that don't have labels in the list will get a higher score.
+	AbsentLabelsPreference []string
+}
+
+// NodeResourcesFitArgs holds arguments used to configure the NodeResourcesFit plugin.
+type NodeResourcesFitArgs struct {
+	// IgnoredResources is the list of resources that NodeResources fit filter
+	// should ignore.
+	IgnoredResources []string
+}
```

Comment:
could you add another field, see [pr](https://github.com/kubernetes/kubernetes/pull/88842)

Reply:
Let's first wait for that PR to get approved before working on this one.